addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -232,6 +233,10 @@ export class Rtc {
 
         browser.addEventListener("pagehide", () => {
             if (this.state.channel) {
+                this.multiTab.broadcastMessage("discuss.rtc.sync", {
+                    channelId: this.state.channel.id,
+                    video: this.state.sendCamera,
+                });
```

Comment:
I would name action `recover`, so the functional meaning is clearer. Syncing is much broader, and is something that should be called for most (all?) rtc state changes

Reply:
The name was confusing, I renamed this event type.